sql/_units.sql

```diff
@@ -86,7 +87,7 @@ CALL apply_correction('_UNITS_devdb_raw', '_manual_corrections', 'classa_prop');
 Using corrected classa, hotel, and otherb unit fields, 
 calculate and then manually correct resid_flag
 */
-DROP TABLE IF EXISTS _UNITS_devdb;
+DROP TABLE IF EXISTS _UNITS_devdb_resid_flag;
```

Comment:
What was the rationale for renaming the table?  Is this necessary or just preference?

Reply:
let me know if this is right @SashaWeinstein I think the `_UNITS_devdb` still exists but since Housing prefer all the units for A2 to be separated from everything else. This intermediate table `_UNITS_devdb_resid_flag` is created to then be split into `_UNITS_devdb` which is everything except A2 and `UNITS_A2_devdb` just A2. @AmandaDoyle

Reply:
Te is correct, this is an intermediate table from which A2 jobs are separated. Adding a new table to pull from allowed me to avoid any cascading changes